assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -515,12 +604,15 @@ describe( 'modules/analytics accounts', () => {
 				);
 			} );
 
-			it( 'does not make a network request if accounts are already present', async () => {
+			it( 'does not fetch from UA properties endpoint if accounts are already present', async () => {
```

Comment:
This has obviously changed a lot now it definitely will be fetching GA4 properties :thinking: 

Fixes
1. Assert that it isn't calling the UA properties endpoint (what I've done)
2. Prefill redux with *all* the network responses and then assert nothing is fetched

It may be that we need to revisit these tests